java/app-encryption/src/main/java/com/godaddy/asherah/appencryption/utils/SafeAutoCloseable.java

```diff
@@ -1,5 +1,8 @@
 package com.godaddy.asherah.appencryption.utils;
 
 public interface SafeAutoCloseable extends AutoCloseable {
+  /**
+   * {@inheritDoc}
+   */
```

Comment:
this is not needed.

Reply:
this is needed since otherwise we get a missing javadoc error